recipes/pdfcombine/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pdfcombine" %}
+{% set version = "0.3.2" %}
+{% set sha256 = "65d57f7024ce51c1f1c9123a1d71085288d5cec145d830b9a145d5c4e335f9ee" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/pdfcombine/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<35]
+
+requirements:
+  host:
+    - python
```

Comment:
When using `noarch` please specify the Python version in both `host` and `run` sections with `python >=3.5`.